src/node/interfaces.cpp

```diff
@@ -1015,16 +1015,12 @@ class BlockTemplateImpl : public BlockTemplate
                 if (current_fees == -1) {
                     current_fees = 0;
                     for (CAmount fee : m_block_template->vTxFees) {
-                        // Skip coinbase
```

Comment:
In commit "interfaces: remove redundant coinbase fee check in `waitNext`" (02d4bc776bbe002ee624ec2c09d7c3f981be1b17)

Might be help to mention #31897 in the commit description to clarify what "now does not include" refers to